config/pacbio_smrt_link_versions.yml

```diff
@@ -149,6 +154,51 @@ default: &default
       versions:
         - v10
         - v11
```

Comment:
I would have a look to see if any of the existing values also apply to v12. https://www.pacb.com/wp-content/uploads/SMRT_Link_User_Guide_v12.0.pdf

Reply:
Just had another look, don't think there's any other fields that are applicable to v12_revio